### ISSUE TITLE
can `node-icns` dev dependency be removed?

### ISSUE THREAD

**Issue by wgwz** [thenewinquiry/bailbloc#52]
[node-icns](https://www.npmjs.com/package/node-icns) seems to deprecated, and unmaintained. any chance we can remove this dependency?

```
warning node-icns@0.0.4: Package no longer supported. Contact support@npmjs.com for more info.
```

**Comment by wgwz** [comment]
i'm just realizing this is probably just being used to resize an icon.. at least that's the place i've seen it used so far (in package.json). probably not a big deal to keep it if it's just being used for that.

**Comment by Artoria2e5** [comment]
Yeah. The `icns` thing is basically windows `ico` files but for macs.

If we really want to get rid of this build dependency, there are newer nodejs packages like [png2icons](https://www.npmjs.com/package/png2icons). Some builds of ImageMagick `convert`, which we currently use for windows ICOs, also comes with icns support.

**Comment by davearch** [comment]
Are the changes acceptable for this issue? I am still new to github pull requests so let me know if there is something else I should do.

**Comment by wgwz** [comment]
hey @davearch, i'm not affiliated with the project in any way, i've only added a few small contributions. but i'd say it's totally acceptable. i opened a MR a while back and had it merged by the developers here. i'm willing to do some code review for you if you'd like, will take a look today.
